deploy/operator/after_upgrade.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+export UPGRADE_ASSISTED_OPERATOR="false"
```

Comment:
Rename to be aligned with the same naming convention
```suggestion
export ASSISTED_UPGRADE_OPERATOR="false"
```

Reply:
Done